js/src/lib.rs

```diff
@@ -28,6 +34,18 @@ fn from_slice_32(bytes: &[u8]) -> [u8; 32] {
     array
 }
 
+#[inline]
+fn from_slice_33(bytes: &[u8]) -> [u8; 33] {
```

Comment:
why not make this sized ref
```suggestion
fn from_slice_33(bytes: &[u8; 33]) -> [u8; 33] {
```

Reply:
data extracted in interfacing functions are `&[u8]`.  We either pre-process it or process inside this function.